### ISSUE TITLE
Toc icon don't show in nav-item responsive screen

### ISSUE THREAD

**Issue by ngtrian** [squidfunk/mkdocs-material#216]
## Description

Sometimes tocs icon in nav-item does not show up
This is my site:
http://docs.nopviet.com/

### Actual behavior

Desktop size:
![image](https://cloud.githubusercontent.com/assets/4327817/23902851/19da4814-08f5-11e7-9082-24d0da4cd974.png)

Responsive size:
![image](https://cloud.githubusercontent.com/assets/4327817/23902892/37e4e242-08f5-11e7-868d-a47ca8a70406.png)

### Package versions

* Python 2.7.12
* mkdocs, version 0.16.1
* Material: 1.3.0

### Project configuration

``` yaml
site_name: nopCommerce Docs
site_url: http://docs.nopviet.com
site_description: Tài liệu trang docs.nopviet.com | nopCommerce Việt Nam
site_author: nopViet Team
repo_url: http://nopviet.com
repo_name: nopViet.com

theme_dir: mkdocs-material/material

extra_css:
  - css/extra.css

extra:
  feature:
    tabs: false
  palette:
    primary: 'indigo'
    accent: 'light blue'
    
google_analytics:
  - 'UA-39056872-8'
  - 'auto'

# Copyright
copyright: 'Copyright &copy; 2017 <a href="http://nopviet.com/" title="nopCommerce Vietnam Community"> nopViet | Cộng đồng nopCommerce Việt Nam</a>'

# Extensions
markdown_extensions:
  - admonition
  - codehilite
  - toc:
      permalink: "#"
      # Xử lý ký tự "đ" khi tạo header id
      slugify: !!python/name:pymdownx.slugs.uslugify
  - pymdownx.arithmatex
  - pymdownx.betterem(smart_enable=all)
  - pymdownx.caret
  - pymdownx.critic
  - pymdownx.inlinehilite
  - pymdownx.magiclink
  - pymdownx.mark
  - pymdownx.smartsymbols
  - pymdownx.superfences
  - pymdownx.tasklist(custom_checkbox=true)
  - pymdownx.tilde

pages:
    - Giới thiệu: index.md
    - Sử dụng nopCommerce: 
      - Bắt đầu: user-guide/bat-dau.md
      - Cài đặt nopCommerce: user-guide/cai-dat-nopcommerce.md
      - Thiết lập cửa hàng:
        - Giao diện trang quản trị: user-guide/giao-dien-quan-tri.md
        - Thiết lập cửa hàng:
          - Thiết lập cơ bản: user-guide/thiet-lap-co-ban.md
          - Thiết lập thanh toán: user-guide/thiet-lap-thanh-toan.md
          - Thiết lập thuế: user-guide/thiet-lap-thue.md
          - Thiết lập giao hàng: user-guide/thiet-lap-giao-hang.md
          - Thiết lập khách hàng: user-guide/thiet-lap-khach-hang.md
          - Tất cả thiết lập (nâng cao): user-guide/thiet-lap-nang-cao.md
        - Quản trị hệ thống: user-guide/quan-tri-he-thong.md
        - Giao diện: user-guide/giao-dien.md
      - Khởi chạy cửa hàng:
        - Quản lý sản phẩm: user-guide/quan-ly-san-pham.md
        - Quản lý đơn hàng: user-guide/quan-ly-don-hang.md
        - Quản lý khách hàng: user-guide/quan-ly-khach-hang.md
        - Lập báo cáo: user-guide/lap-bao-cao.md
      - Tiếp thị cửa hàng:
        - Công cụ quảng cáo: user-guide/cong-cu-quang-cao.md
        - Quản lý nội dung: user-guide/quan-ly-noi-dung.md
      - Tài liệu hướng dẫn cho các version cũ: user-guide/tai-lieu-nopcommerce-khac.md
    - Phát triển nopCommerce: 
      - Tut hướng dẫn phát triển: developer-guide/tut-developer.md
      - Tạo plugin nopCommerce:
        - Cách viết plugin nopCommerce: developer-guide/cach-viet-plugin-nopcommerce.md
        - Cách viết phương thức thanh toán riêng: developer-guide/cach-viet-phuong-thuc-thanh-toan-rieng.md
        - Cách viết phương thức giao hàng riêng: developer-guide/cach-viet-phuong-thuc-giao-hang-rieng.md
        - Plugin với truy cập dữ liệu: developer-guide/plugin-voi-truy-cap-du-lieu.md
        - Cách tạo menu cho plugin trong trang quản trị: developer-guide/cach-tao-menu-plugin.md
        - Thêm CSS và JS vào plugin nopCommerce: developer-guide/them-css-va-js-vao-plugin.md
        - Video hướng dẫn tạo plugin: developer-guide/video-huong-dan-tao-plugin.md
        - Khóa học về phát triển plugin nopCommerce: developer-guide/khoa-hoc-phat-trien-plugin.md
      - Hướng dẫn thiết kế: developer-guide/huong-dan-thiet-ke.md
    - Các câu hỏi thường gặp (FAQ): faq-nopcommerce.md
```

### System information

* OS: Ubuntu Server 16.04.2 LTS
* Browser: Chrome

**Comment by squidfunk** [comment]
To clarify - you're talking about the mobile view?

**Comment by ngtrian** [comment]
Yes, It seems that the mobile view don't show toc icon

**Comment by squidfunk** [comment]
I can confirm this in your docs but not in mine - can you narrow the error down?

**Comment by ngtrian** [comment]
I have found a problem. If md file content does not contain header 1 (Which has header 2, header 3..), then icon toc will not be displayed in mobile view . Although header 1 is generated automatically from the setting pages in mkdocs.yml file.

**Comment by squidfunk** [comment]
Ok perfect, I will submit a fix.

**Comment by squidfunk** [comment]
Fixed in 1.4.0 which was just released.
